style.css

```diff
@@ -318,4 +337,22 @@ ul#contributors .links {
   display: flex;
   justify-content: center;
   gap: .25rem;
-}
\ No newline at end of file
+}
+/* .row{
```

Comment:
@Devemmy01 kindly clean this part(commented code blocks) up

Reply:
I'll do that now